pandas_gbq/gbq.py

```diff
@@ -774,6 +775,8 @@ def read_gbq(
     col_order : list(str), optional
         List of BigQuery column names in the desired order for results
         DataFrame.
+    columns : list(str), optional
+        Alias for col_order
```

Comment:
Let's flip this. Add the description for col_order here and the description of col_order should be: Alias for columns, retained for backwards compatibility.